spec/embedded_sass.proto

```diff
@@ -192,6 +192,9 @@ message InboundMessage {
       // `null` instead.
       string error = 3;
     }
+
+    // Whether `containing_url` in `CanonicalizeRequest` is unused.
+    bool containing_url_unused = 4;
```

Comment:
In protobuf, `bool` default to `false` when it's unset.

The default `unused = false` is effectively `accessed = true`, which means `cacheable = false`. This is the ideal default, that it works correctly even if the host does not implement the tracking, making the protocol backward compatible.

Reply:
Can you mention this reasoning in the documentation?

Reply:
Added. Please feel free to edit this PR if you would like to rephrase the language.